src/bosh_azure_cpi/spec/unit/utils/bosh_agent_util_spec.rb

```diff
@@ -33,77 +32,102 @@
     allow(config).to receive(:agent).and_return({'mbus' => 'http://u:p@somewhere'})
   end
 
-  context 'when using registry' do
-    let(:uses_registry) { true }
-    let(:expected_user_data) do
-      {
-          registry: { endpoint: registry_endpoint },
-          server: { name: instance_id },
-          dns: { nameserver: dns }
-      }
-    end
+  # TODO: coverage: need to implement the following specs to increase coverage
```

Comment:
Are the TODOs still required?

Reply:
Yes, these TODOs make explicit some areas of un-covered code that are already in `master`, and are technically unrelated to this PR (other than the fact that they bing coverage down for this and all other PRs until they're fixed and coverage returns to 100%, which is stated as a requirement in the template for all PRs).

Reply:
FYI: More info on these TODOs is in #652.